wiki/Media_use_between_lack_of_control..._2775579.qmd

```diff
@@ -0,0 +1,28 @@
+---
+title: "Binge Watching and Psychological Well-Being: Media Use between Lack of Control and Perceived Autonomy"
+subtitle: "BA New Media / Online Communication KF/BF A: Spring Term 2023"
+author: "Niels Iancu"
+categories:
+  - "Media Psychology"
+  - "Digital Media"
+  - "Well Being"
+---
+
+KARQ-Questions:
+
+(K) "Binge watching provides new opportunities for self-determined forms of entertainment consumption and can positively influence media enjoyment and well-being through perceived autonomy"(Granow et al., 2018, p.9).
+
+(A) The article states that through different media experiences, especially binge-watching, your well-being is positively affected. This relates to the fact that you have an increase in your perceived autonomy through the media use of binge-watching. So an increase in perceived autonomy means an increase in your personal well-being. Self-determination and with that autonomy is an important factor in the relationship between media reception, media enjoyment and your psychological state.
+
+(R) The articles relating to digital media use and autonomy have very similar arguments for the most part. Other articles describe that an increase of choice leads to an increase in perceived autonomy (Schneider et al., 2018, p.171). This correlates to the main article of Granow et al., which states that the choice of being able to select your media content on your own leads to an increase of perceive autonomy. But not all articles are that positive in relation to autonomy and digital media use. They argue that through the control of your attention digital media interferes with your autonomy. This is obviously a wanted effect for social media networks and they use different tools to gain and keep that attention, so this effect of decrease in your perceived autonomy follows naturally (Sahebi & Formosa, 2022).
+
+(Q) 1.Consuming digital media, in this article binge-watching, is said to have a positive impact on your perceived autonomy. But don't you lose part of your autonomy while being stuck in this episode of binge-watching?
+2.Does the content that you're watching during a binge-watching phase impact your perceived autonomy, for example, does a TV show that's very thrilling gain more of your attention and through that has more impact on your autonomy?
```

Comment:
Good question. Try to put together some arguments for why or why not this should be the case.